src/app/content/components/Page/contentLinkHandler.ts

```diff
@@ -126,12 +129,14 @@ export const contentLinkHandler = (anchor: HTMLAnchorElement, getProps: () => Co
       }, {hash, search: searchString}));
     } else {
       // defer to allow other handlers to execute before nav happens
+      const bookParams = (await getBookPageUrlAndParams(book, page, bookConfigLoader)).params;
+      const bookState = (await getBookPageUrlAndParams(book, page, bookConfigLoader)).params;
```

Comment:
why are you running the same query twice?

Reply:
oops mistake.